source/_components/switch.mqtt_template.markdown

```diff
@@ -0,0 +1,117 @@
+---
+layout: page
+title: "MQTT Template Switch"
+description: "Instructions on how to integrate MQTT Template switches into Home Assistant."
+date: 2018-04-09 23:38
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: mqtt.png
+ha_category: Switch
+ha_release: 0.68
+ha_iot_class: depends
+---
+
+The `mqtt_template` switch platform lets you control your MQTT enabled switches.
+
+This platform is similar to the [switch.mqtt](/components/switch.mqtt) platform, only it allows the usage of a template as the on/off payload.
+
+To enable this switch in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yml entry
+switch:
+  - platform: mqtt_template
+    command_topic: "home/bedroom/switch1/set"
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.